learn/appchains/runtime-features.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Core Runtime Features
+description: Learn the high-level definitions of how a Tanssi Appchain works
+---
+
+# Core Runtime Features {: #core-runtime-features }
+
+## Introduction {: #introduction}
```

Comment:
This introduction could be improved a bit by, perhaps, briefly mentioning some of the topics covered in the following sections